src/Client.php

```diff
@@ -25,295 +26,233 @@ class Client
      * Creates a new client instance with an API Key and API Secret
      * You can obtain these from the Communicacie, who will register your app in Lassie
      *
-     * @param string $key
-     * @param string $secret
+     * @param string $token static access token for Directus
      */
-    public function __construct(string $key, string $secret)
+    public function __construct(string $token, string $base_url = "https://admin.eshdavinci.nl")
     {
         $stack = new HandlerStack();
         $stack->setHandler(new CurlHandler());
-        $stack->push(KeyMiddleware::addData($key, $secret));
         $this->guzzleClient = new GuzzleClient([
-        'base_uri' => 'https://davinci.lassie.cloud/',
-        'timeout'  => 30.0,
-        'handler' => $stack
-    ]);
+            "base_uri" => $base_url,
+            "timeout" => 4.0,
```

Comment:
Why the lower timeout? Although 5 sec should be enough

Reply:
No strong reason in particular. I don't really like too long timeouts... Usually if the server doesn't respond within 4 seconds, it won't respond at all.